ui-components/src/overlays/Modal.tsx

```diff
@@ -77,7 +73,7 @@ export const Modal = (props: ModalProps) => {
             onClick={props.onClose}
             tabIndex={0}
           >
-            <Icon size="medium" symbol="close" />
+            <Icon size="medium" symbol="close" fill={IconFillColors.primary} />
```

Comment:
I wonder if we shouldn't hardcode the primary color being the color of the close icon 🤔